paasta_tools/long_running_service_tools.py

```diff
@@ -119,6 +119,28 @@ def __init__(
             soa_dir=soa_dir,
         )
 
+    def get_bounce_method(self) -> str:
+        raise NotImplementedError
+
+    def get_kubernetes_namespace(self) -> str:
```

Comment:
Yeah, this is kinda nasty isn't it. Do you think we need `kubernetes_long_running_service_tools.py`?

And then we have for k8s deployments:
`InstanceConfig -> LongRunningServiceConfig -> KubernetesLongRunningService -> KubernetesDeploymentConfig`

and for cassandra/flink etc
`InstanceConfig -> LongRunningServiceConfig -> KubernetesLongRunningService -> CassandraClusterDeploymentConfig`

Reply:
this is all so weird.. why can't we just treat these configs as open maps and do `self.config.get()` instead of having to declare these elaborate hierarchies of getters and setters. i'm not a fan of adding another link to the chain there. if we can with low effort make kubernetes client take care of the namespace (see followup comment in another thread) - i'd prefer that. if not - then fine i guess..

Reply:
I think this is fine. At the end, LongRunningServiceConfig is for Kubernetes only. Personally I don't like to add another link either.